sources/lib/run-time/posix-threads.c

```diff
@@ -512,8 +506,6 @@ dylan_value primitive_make_thread(dylan_value t, dylan_value f, DBOOL s)
   thread->handle1 = 0;       // runtime thread flags
   thread->handle2 = rthread; // runtime thread object
 
-  // param.sched_priority = priority_map(priority);
-
```

Comment:
This should get implemented not deleted.

Reply:
Not sure how to proceed on this one. As far as I can see POSIX doesn't allow you to set the 'nice' of a thread (though Linux does) - in which case we'd have to go for the realtime schedulers (SCHED_FIFO or SCHED_RR). Would these require a privileged process to run? And, Macos doesn't have `pthread_setschedparam` at all, or at least some versions don't.
So maybe a Linux only implementation is the best way?

Reply:
No reason you need to implement this as part of this pull request. Just undelete those lines for now?

Reply:
OK I've reverted the change to that file now.